spec/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'simplecov'
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
-SimpleCov.minimum_coverage 98.96
+SimpleCov.minimum_coverage 98.91
```

Comment:
I'm creating a trello ticket to bump this up to 100%. We've got a good thing going having those at 100% coverage. Don't worry about this one, it's a bit trickier.

Reply:
Thanks @StupidCodeFactory